testing/tests/DevExpress.ui.widgets.form/form.markup.tests.js

```diff
@@ -773,6 +776,197 @@ QUnit.module('Form', () => {
     });
 });
 
+QUnit.module(`"${TOGGLE_CONTROLS_PADDING_CLASS}" class`, ()=>{
+    ['dxCheckBox', 'dxSwitch', 'dxRadioGroup'].forEach(editorType => {
+        const componentName = editorType.split('dx')[1].toLowerCase();
+
+        test(`${editorType} should have class when labelLocation=top, label.alignment=default (default=left), label.visible=true (T1126956)`, function(assert) {
+            const $form = $('#form').dxForm({
+                labelLocation: 'top',
+                items: [
+                    {
+                        itemType: 'group',
+                        items: [
+                            {
+                                itemType: 'group',
+                                items: [{
+                                    dataField: editorType,
+                                    label: { visible: true },
+                                    editorType,
+                                }],
+                            },
+                        ],
+                    },
+                ],
+            });
+            const $componentWrapper = $form.find(`.dx-${componentName}`).parent();
+
+            assert.strictEqual($componentWrapper.hasClass(TOGGLE_CONTROLS_PADDING_CLASS), true);
+        });
+
+        test(`${editorType} should have class after visibility change to true (labelLocation=top, label.alignment=left)`, function(assert) {
+            const $formContainer = $('#form').dxForm({
+                labelLocation: 'top',
+                items: [
+                    {
+                        itemType: 'group',
+                        items: [
+                            {
+                                itemType: 'group',
+                                items: [{
+                                    dataField: editorType,
+                                    label: { visible: false },
+                                    editorType,
+                                }],
+                            },
+                        ],
+                    },
+                ],
+            });
+            const optionLabelVisible = 'items[0].items[0].items[0].label.visible';
```

Comment:
What did u want to say with this name?
```suggestion
            const labelVisibleOptionName = 'items[0].items[0].items[0].label.visible';
```